helix-core/src/test/java/org/apache/helix/common/ZkTestBase.java

```diff
@@ -711,6 +711,7 @@ protected Message createMessage(Message.MessageType type, String msgId, String f
 
   @AfterClass
```

Comment:
nit, I believe you can define multiple AfterClass methods so we don't need to mix things up.

Reply:
We need this one to be called last after all test class level cleaning up. Defining another @afterClass in base class, we can't guarantee this order.